src/components/stats/StatsGraph.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+
+import d3Graph from '../../utils/d3Graph';
+
+class StatsGraph extends Component {
+  state = {
+    filter: 'all',
+  }
+
+  fetchIfChanged = (_from, _to) => {
+    const { statsFrom, statsTo, fetchStatsByRange } = this.props;
+    if (_from !== statsFrom || _to !== statsTo) {
+      fetchStatsByRange(_from, _to);
+    }
+  }
+
+  getGraphState() {
+    const { filter } = this.state;
+    const { statsByDate } = this.props;
+
+    if (statsByDate && statsByDate.length) {
+      const data = this.props.statsByDate.map(function(d) {
+        d.value = 0;
+        if (d.edit && (filter === 'all' || filter === 'edited')) d.value += d.edit;
```

Comment:
Given this small distinction between filter values and data values, would it clean up this code significantly to rename the filters to be exactly the same as the data values, so that you could reference directly?

Additionally, if the default value is `0`, avoiding its addition won't change the output of the method.

For instance, something like this might work (untested code):

```js
const includedFields = (filter === 'all') ?
  ['edit', 'fixed', 'skip', 'noterror'] : [filter];
d.value = includedFields.reduce((sum, field) => sum + d[field], 0);
```